workloads/scale-perf/README.md

```diff
@@ -56,21 +56,37 @@ Default: `false`
 Enable/Disable the ability to compare two runs. If set to `true`, the next set of environment variables pertaining to the type of test are required
 
 ### BASELINE_CLOUD_NAME
-Default: ``    
```

Comment:
@morenod can we leave the default values or was there a reson you removed them?

Reply:
Empty '' is breaking the .md file generation. If default is none, there is no default

Reply:
restored empty defaults